pkg/watcher/skr_webhook_manifest_manager.go

```diff
@@ -23,40 +23,39 @@ import (
 	"github.com/kyma-project/lifecycle-manager/pkg/util"
 )
 
-// SKRWebhookManifestManager is a SKRWebhookManager implementation that applies
+// WatcherManifestManager is a SKRWebhookManager implementation that applies
```

Comment:
Without change the skr-webhook service name, this renaming is a bit misleading tbh, but please then enhance the description comment of this WatcherManifestManager, it still contains SKRWebhookManager in the context.

Reply:
I revert the naming changes. Can be done in a follow-up and more thorougly 👍